.github/scripts/status-check/services/cli-service.js

```diff
@@ -32,6 +30,7 @@ class CLIService {
   }
 
   async createIssue(issueBody) {
+    // exadel-inc/esl-core-team, 
```

Comment:
Proposal to define assignee via passing into service constructor and store config in separate file